pkg/controller/handler.go

```diff
@@ -137,6 +138,43 @@ func (h *handler) CreateServiceInstance(in *servicecatalog.Instance) (*serviceca
 	return h.apiClient.Instances(in.ObjectMeta.Namespace).Update(in)
 }
 
+func (h *handler) DeleteServiceBinding(sb *servicecatalog.Binding) error {
+	// this logic to set and update the timestamp is TPR specific. to be moved to the API server
+	dts := metav1.Now()
+	sb.DeletionTimestamp = &dts
```

Comment:
I can see this is where the timestamp is created. Which of the calls below actually persist it to storage (tpr or otherwise)? Is it the `.Update(sb)` calls?